### PR TITLE
Span activesupport 3.x AND 4.x

### PR DESCRIPTION
**Pull request by atz**
Add travis config for both and two different ruby versions.
@wmene @jmartin-sul

### CODE REVIEW

**Comment by atz** [comment]
Build passed, now against 2 versions of ruby.  Not sure it is picking up on the `gemfiles` directive though... hrmmm.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 language: ruby
 rvm:
+  - 1.9.3
   - 2.1.0
-
+gemfiles:
```
Comment (jpstroop) [inline .travis.yml]: @atz I think it might be singular, i.e. `gemfile:`. Cf. http://docs.travis-ci.com/user/build-configuration/#The-Build-Matrix

Comment (atz) [inline .travis.yml]: Ah nice.  The conventional directory is `gemfiles` and the config is `gemfile`.  Got it.

**Comment by atz** [comment]
Ok, corrected and rebased.

**Comment by jpstroop** [comment]
Werd. I good, you OK?

**Comment by atz** [comment]
Yep, please pull and publish at your convenience.

**Comment by jpstroop** [comment]
0.0.3: https://rubygems.org/gems/osullivan

**Comment by wmene** [comment]
My fault.  I should've used the >= operator

**Pull request merged**